crlcore/src/ccore/Utilities.cpp

```diff
@@ -256,26 +257,25 @@ namespace CRL {
     if (coriolis_top_env) {
       coriolis_top = coriolis_top_env;
     } else {
-      //We're running as a binary and PyCRL hasn't been instantiated
-      // use python to find our where we should look
-      using namespace boost::process;
-      std::error_code ec;
-      ipstream out;
-      //TODO: will need fixing for windoews
-      system("python3 -c 'from coriolis import CRL; import os; print(os.path.abspath(os.path.dirname(CRL.__file__)))'", std_out > out, ec); 
-      if (ec) {
-          cerr << "Unable to find coriolis python package. Please set CORIOLIS_TOP environment variable\n";
-          exit ( 1 );
+      std::error_code     rvalue;
```

Comment:
I think leave comment in.

Reply:
OK. Will put them back. What I really should do is write a comprehensive documentation about the Coriolis initialization process especially the self location.